nemo_rl/algorithms/dpo.py

```diff
@@ -265,6 +270,36 @@ def add_ref_logprobs_to_data(dataloader, policy, master_config, is_val=False):
 # Training & Validation
 # =======================================================
 def validate(
+    policy: PolicyInterface,
+    val_dataloader: StatefulDataLoader | dict[str, StatefulDataLoader],
+    tokenizer,
+    loss_fn,
+    step: int,
+    master_config: MasterConfig,
+    val_batches: int,
+    val_batch_size: int,
+    val_mbs: int,
+    logger: Logger,
+):
+    val_metrics, validation_timings = {}, {}
+    for k, v in val_dataloader.items():
+        k_val_metrics, k_validation_timings = validate_one_dataset(policy, v, tokenizer, loss_fn, step, master_config, val_batches, val_batch_size, val_mbs, k)
+        if k == "validation":
+            prefix = "val"
+        else:
+            prefix = f"{k}-val"
+
+        logger.log_metrics(k_val_metrics, step, prefix=prefix)
+        logger.log_metrics(k_validation_timings, step, prefix=f"timing/{prefix}")
+
+        val_metrics[prefix+"_loss"] = k_val_metrics["val_loss"]
+        val_metrics[prefix+"_accuracy"] = k_val_metrics["accuracy"]
+        validation_timings[prefix+"_total_validation_time"] = k_validation_timings["total_validation_time"]
```

Comment:
Suggestion to keep naming consistent (NB: same for RM)
```suggestion
        validation_timings[prefix+"_total_val_time"] = k_validation_timings["total_validation_time"]
```

Reply:
Yes, I updated